tests/functional/test_loaded_model.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+import grpc
+import pytest
+
+from tritonclient.grpc import service_pb2
+from tritonclient.grpc import service_pb2_grpc
+import tritonclient.grpc as grpcclient
+from model_config import MODEL_CONFIG
+PORT=8111
+
+
+@pytest.mark.parametrize("model", MODEL_CONFIG.keys())
+def test_inference(model):
+    triton_client = grpcclient.InferenceServerClient(
+        url=f"localhost:{PORT}",
+        verbose=False)
+
+    config = MODEL_CONFIG[model]
+
+    inputs = []
+    inputs.append(grpcclient.InferInput(config["input_name"], config["shape"], "FP32"))
+
+    inputs[0].set_data_from_numpy(np.ones(config["shape"], dtype=np.float32))
```

Comment:
these lines regarding the preparation of inputs can be enclosed in one function (to use it later in all tests) and taken as parameters e.g. config, shape and datatype